### DISCUSSION TITLE
Is missed `objc_imageinfo` section when compiling Objective-C code with r25 LTS intended behavior or not?

### DISCUSSION THREAD

**Discussion by Van-der-Deken** [android/ndk#1777] [Q&A] [answered]
Hello,
Currently we use r23 LTS version of NDK and all is fine - among sections in result binaries with Objective-C code `objc_imageinfo` section also present (as in previous versions). I tried to check compatibility with current LTS - r25 and got failure due to absence of it. We use this section for the same purpose as Apple on their platform - to check existence of Objective-C code in the binary. It's not a complex issue to overcome on our side, as other more important sections still present, but I want to understand first is this a bug or not?

**Comment by DanAlbert** [comment, marked as answer]
The NDK doesn't support objective C, so there is no intended behavior.
